### ISSUE TITLE
Poor and worsening keyboard usability

### ISSUE THREAD

**Issue by ndarilek** [rancher/rancher#3674]
Version: 0.59.0

Steps: 
  1. Disconnect your mouse, throw it out the window, etc.
  2. Try performing just about any task using just the keyboard:
    * Try to select an environment other than the default.
    * You can click on one of the unlabeled icons to pop up the menu for an app/stack, but try to then tab to and activate one of the controls, like Edit.
    * Try to add a port in a load balancer and set it to HTTP rather than TCP.

Results: 
  Sadness and woe. As a blind user, you went from having a somewhat keyboard-friendly interface to one that barely works and requires lots of hackery, and I'm having a harder and harder time administering my Rancher ecosystem.

Expected:
  I should be able to use the UI via the keyboard, without needing my screen reader to simulate mouse clicks.

See also: #3094

**Comment by tfiduccia** [comment]
I still can't select a different environment.  to reproduce:
1.  Add a couple of environments
2.  Try to use keyboard to select environments

**Comment by lvuch** [comment]
@tfiduccia are you able to tab to the environment dropdown and then again press tab to select "manage environments" ?

**Comment by ndarilek** [comment]
That much works, but how would I switch environments via the keyboard?

**Comment by lvuch** [comment]
https://github.com/rancher/ui/pull/526 fixes tab nav in environment dropdown and action menus

**Comment by lvuch** [comment]
ability to tab open action menu resolved in https://github.com/rancher/ui/pull/538 @westlywright will add keypress

**Comment by lvuch** [comment]
@westlywright enter works, click works. but when using enter on keyboard menu lingers.

**Comment by tfiduccia** [comment]
Version - master 3/20
Not fixed - can't tab to menu item and in containers the down arrow does not work

**Comment by tfiduccia** [comment]
Version - master 3/22
Verified fixed

**Comment by ndarilek** [comment]
Just wanted to thank you all for your work on this! I've spent a few minutes with it thus far and it's so much more pleasant to work with. I'd set up Rancher a while back, and honestly it was starting to bug me to the point that I'd considered replacing it. But so far I've been able to switch environments and access service controls very easily, and can start doing things quickly and efficiently again. :) So thanks for taking this seriously.
